NewForce-at-Mountwest/bangazon-site-take-2-centaurs
# Add Payment Type to Order

User should be able to add a payment type to their open order.

To test:
-Launch application
-Attempt to navigate to ~/orders/edit/1 without logging in and check for redirect to login
-Log in to application
-Navigate to ~/orders/edit/1 and select a payment type, then click submit
-User should be redirected to index with orders shown, payment type and date completed fields populated.
